graphql/util.go

```diff
@@ -767,10 +767,21 @@ func ModifyVersionHandler(ctx context.Context, dataConnector data.Connector, pat
 			}
 			if p.IsParent() {
 				for _, childPatchId := range p.Triggers.ChildPatches {
-					err = ModifyVersionHandler(ctx, dataConnector, childPatchId, modifications)
+					p, err := patch.FindOneId(childPatchId)
```

Comment:
can we get all of the patches with one query and then iterate through the patches instead?

Reply:
That's a good point. There is another place where we do this so I will create another ticket to write a getchildPatches query and use it where appropriate.